packages/components/src/Actions/ActionButton/ActionButton.test.js

```diff
@@ -110,6 +110,25 @@ describe('Action', () => {
 		expect(wrapper.type()).toBe(null);
 	});
 
+	it('should render a button without an overlay component if inProgress is true', () => {
+		function OverlayComponent() {
+			return <div>OverlayComponent</div>;
+		}
+
+		const props = {
+			...myAction,
+			inProgress: true,
+			overlayComponent: OverlayComponent,
+			overlayPlacement: 'bottom',
+		};
+
+		// when
+		const wrapper = shallow(<ActionButton {...props} />);
+
+		// then
+		expect(wrapper.getNode()).toMatchSnapshot();
```

Comment:
snapshot are not read today because we have too much.  instead expect you try to find it but not found.